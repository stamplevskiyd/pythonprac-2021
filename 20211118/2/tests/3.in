class C:
        num = Num()
print(C().num)
c = C()
c.num = 1.23456
print(c.num)
c.num = [i % 2 == 4 for i in range(10)]
print(c.num)
