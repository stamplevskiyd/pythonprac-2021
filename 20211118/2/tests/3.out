0
1.23456
10
